FAQ.md

```diff
@@ -0,0 +1,111 @@
+# Frequently Asked Questions
+
+## Where can I find upgrades?
+https://origin-release.svc.ci.openshift.org/
+
+Each Preview/Beta version is marked with an asterix *
```

Comment:
asterisk means the build won't be pruned. We may want to keep arbitrary nightlies, regardless of Beta status.

I don't think its worth mentioning, its release-controller detail

Reply:
I made a hint out of this comment. I stumbled of that. If it's resolved that nightlies won't be deleted I think this comment can be removed. It's on you to decide that.